### PR TITLE
as_draws()

### LINKED ISSUE #532: Implement `posterior::as_draws()` for cmdstanr objects
Per the conversation at https://github.com/stan-dev/posterior/issues/183, now that {posterior} is on CRAN it would be wonderful if cmdstanr's objects could have an implementation of `posterior::as_draws()` so that users would not have to remember what function to call to get draws objects out of models when moving between packages. Perhaps this could be a simple wrapper around the `$draws()` method? Thanks!

### PR DESCRIPTION
**Pull request by jgabry**
#### Submission Checklist

- [x] Run unit tests
- [x] Declare copyright holder and agree to license (see below)

#### Summary

Closes #532 

Adds `as_draws()` methods for CmdStanR's fitted model objects. Since this is just a wrapper around `fit$draws()`, the only argument is `...`, which is passed to `fit$draws()` (so can include `variables`, `inc_warmup`, `format`, etc.). 

#### Copyright and Licensing

Please list the copyright holder for the work you are submitting 
(this will be you or your assignee, such as a university or company): 
**Columbia University**

By submitting this pull request, the copyright holder is agreeing to 
license the submitted work under the following licenses:

- Code: BSD 3-clause (https://opensource.org/licenses/BSD-3-Clause)    
- Documentation: CC-BY 4.0 (https://creativecommons.org/licenses/by/4.0/)

### CODE REVIEW

File: R/fit.R [hunk 1]
```diff
@@ -1549,3 +1549,56 @@ CmdStanDiagnose$set("public", name = "save_output_files", value = save_output_fi
...
+
+
+# as_draws ----------------------------------------------------------------
+#' Create a `draws` object from a CmdStanR fitted model object
+#'
+#' Create a `draws` object supported by the \pkg{posterior} package. These
+#' methods are just wrappers around CmdStanR's [`$draws()`][fit-method-draws]
+#' method provided for convenience.
+#'
+#' @aliases as_draws
```
Comment (rok-cesnovar) [inline R/fit.R:1562]: Do we mention that users can now do things like

```
model %>% as_draws_XXX()
```

or is that for a vignette with an example?

Comment (jgabry) [reply R/fit.R:1562]: In the `@examples` section of the doc I have this:  

```
#' # posterior's as_draws_*() methods will also work
#' posterior::as_draws_rvars(fit)
#' posterior::as_draws_list(fit)
```

Do you think that's sufficient?

Comment (rok-cesnovar) [reply R/fit.R:1562]: Yeah, should be good.

**Review by rok-cesnovar — APPROVED** [review]
Looks good to me. Merge once tests pass.

**Pull request merged**